### PR TITLE
Responsive Design: Next-web-page-clone (Milestone 2)

### PR DESCRIPTION
**Pull request by felix-vondee**
In this project, we learned how to build a responsive web page with CSS media queries.
Objectives & Design decisions:

We used media query cut off points at 768px and 1024px. This allowed us to add media queries for mobile, tablet, and desktop layouts.
We handled all resizing of elements on the page by editing their properties inside the media query selectors.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**You've done pretty well here with a good PR title and description, and you obviously paid attention to the project details. 👌**

However, let's make a couple of changes.

- [x] Kindly fix the stylelint issues in the project the stylelint linter check is failing, please fix it.

- Please refer to other comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,724 @@
...
+        <div class="section2_items">
+            <img alt="image"
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/california-car-ev-hydrogen-future-electruc-fcev-ev-480x270.png">
+            <div>
+                <a class="section2_links" href="">Want to drive a hydrogen EV in the US? Move to California</a>
+                <div>
+                    <a class="special_text" href="#">Matthew Beedham</a>
+                    <p class="special_text">· 22 minutes ago</p>
+                </div>
+            </div>
```
Comment (mikenath223) [inline index.html]: Each article in the Latest News section should have an image and a title (h1--h6) tag, both of them with a link (a tag). i.e.

```
<h3>
  <a href="#" class="img-text">
    <p> Want to drive a hydrogen EV in the US? Move to California</p>
  </a>
</h3>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,724 @@
...
+    <section id="section5">
+        <h2 class="none">hello</h2>
+        <div class="section5_items">
+            <img src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp2.stackassets.com/7a5a98dff30d5c38ecbc1700812eb2e1b93dce93/store/4b1f63289bd195a7bbafafbe36579fe597a5f1e7a0cb8165012908f3db74/sale_28088_primary_image.jpg"
+                alt="">
+            <div>
+                <p><a href="#"> The Ultimate 2021 White Hat Hacker Certification Bundle</a></p>
+                <span class="original_price">$1345</span>
+                <span class="new_price">$39.90</span>
+            </div>
```
Comment (mikenath223) [inline index.html]: Each article in the Latest Deals section should have an image and a title (h1--h6) tag, both of them with a link (a tag). i.e.

```
h4>
  <a href="#" class="img-text">
    <p> The Ultimate 2021 White Hat Hacker Certification Bundle</p>
  </a>
</h4>
```

File: index.html (hunk 1 above)
Comment (felix-vondee) [reply index.html]: Thanks for the suggestions. These changes have been made

File: index.html (hunk 2 above)
Comment (felix-vondee) [reply index.html]: Thanks for the suggestions. These changes have been made

**Review by SmithVyne — APPROVED** [review]
### Project Approved 😄 🏅 🏅 
Congratulations your project is approved. You can now merge this branch and move on in the curriculum.

You can also reach out to me on slack, **@SmithVyne** if you have any other questions.

**Pull request merged**